rest-api-spec/src/main/resources/rest-api-spec/api/indices.simulate_index_template.json

```
@@ -0,0 +1,35 @@
+{
+  "indices.simulate_index_template":{
+    "documentation":{
+      "url":"https://www.elastic.co/guide/en/elasticsearch/reference/master/indices-templates.html",
+      "description": "Simulate matching the given index name against the index templates in the system"
+    },
+    "stability":"stable",
+    "url":{
+      "paths":[
+        {
+          "path":"/_index_template/_simulate_index/{name}",
+          "methods":[
+            "POST"
+          ],
+          "parts":{
+            "name":{
+              "type":"string",
+              "description":"The name of the index (it will have to be a concrete index name)"
```

Minor wording change
```suggestion
              "description":"The name of the index (it must be a concrete index name)"
```